pkg/cmd/stack/deploy.go

```diff
@@ -236,3 +237,15 @@ func waitForPodsToBeRunning(ctx context.Context, s *model.Stack, c *kubernetes.C
 	}
 	return fmt.Errorf("kubernetes is taking too long to create your stack. Please check for errors and try again")
 }
+
+func DisplayWarnings(s *model.Stack) {
+	if len(s.Warnings) > 0 {
+		if len(s.Warnings) == 1 {
+			log.Warning("'%s' field is not currently supported and will be ignored.", s.Warnings[0])
+		} else {
+			notSupportedFields := strings.Join(model.GroupWarningsBySvc(s.Warnings), "\n  - ")
```

Comment:
Let's create a new field for volume mount warning, so they are also displayed after building the images